src/account-info/AccountDetailsScreen.js

```diff
@@ -16,17 +16,21 @@ class AccountDetailsScreen extends Component {
     avatarUrl: string,
   }
 
+  componentWillMount() {
+    this.email = this.props.email;
+  }
+
   render() {
-    const { auth, email, users, fetchMessages, popRoute } = this.props;
-    const { fullName, avatarUrl, status } = users.find(x => x.email === email) || { fullName: 'A', avatarUrl: '', status: 'unknown' };
+    const { auth, users, fetchMessages, popRoute } = this.props;
+    const { fullName, avatarUrl, status } = users.find(x => x.email === this.email) || { fullName: 'A', avatarUrl: '', status: 'unknown' };
```

Comment:
We should treat a missing user here as an error instead of showing 'A' with status 'unknown'